MQ2AutoSize.cpp

```diff
@@ -225,10 +382,13 @@ void ChangeSize(PlayerClient* pChangeSpawn, float fNewSize)
 
 void SizePasser(PSPAWNINFO pSpawn, bool bReset)
 {
-	if ((!bReset && !AS_Config.OptByZone && !AS_Config.OptByRange) || GetGameState() != GAMESTATE_INGAME) return;
-	PSPAWNINFO pChSpawn = (PSPAWNINFO)pCharSpawn;
```

Comment:
Unsure if this is intentional, but pCharSpawn is pControlledPlayer, not pLocalPlayer.

pControlledPlayer is the player that you're currently in control of. So really this will be different for an eye of zomm, or maybe a mount.

Reply:
The original plugin did not resize a controlled spawn like Eye of Zomm. Mounts are handled in a different way successfully in both the original and the update here.

I've tinkered with trying to get Eye of Zomm to resize using pControlledPlayer and have been unsuccessful.